test/parser/test/calc.hvml

```diff
@@ -110,10 +110,12 @@
     </body>
 
     <init as="params">
-        { "c_text" : "0" } //计算器液晶屏上显示的字符串内容
-        { "l_value" : "" } //左值，以字符串形式暂存
-        { "r_value" : "" } //右值，以字符串形式暂存
-        { "operator" : "" } //当前操作符，选项有+-*/%，""表示未设置
+        [
+            { "c_text" : "0" },  //计算器液晶屏上显示的字符串内容
+            { "l_value" : "" },  //左值，以字符串形式暂存
+            { "r_value" : "" },  //右值，以字符串形式暂存
+            { "operator" : "" }, //当前操作符，选项有+-*/%，""表示未设置
+        ]
```

Comment:
in this way, codes in hvml might use '$params[2].r_value' to access r_value, but if u fall back to {...},
this can be simply accessed with '$params.r_value'.